dotCMS/src/main/java/com/dotcms/rest/api/v1/content/ContentRelationshipsHelper.java

```diff
@@ -0,0 +1,186 @@
+package com.dotcms.rest.api.v1.content;
+
+import com.dotcms.rest.RESTParams;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.contentlet.business.ContentletAPI;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.languagesmanager.business.LanguageAPI;
+import com.dotmarketing.portlets.structure.model.ContentletRelationships;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UtilMethods;
+import com.dotmarketing.util.json.JSONArray;
+import com.dotmarketing.util.json.JSONException;
+import com.dotmarketing.util.json.JSONObject;
+import com.dotmarketing.viewtools.content.util.ContentUtils;
+import com.liferay.portal.model.User;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Provides the utility methods that interact with content relationships in dotCMS. These methods
+ * are used by the Content Relationships REST end-point.
+ *
+ * @author Will Ezell
+ * @author Jose Castro
+ * @version 4.2
+ * @since Oct 11, 2017
+ */
+public class ContentRelationshipsHelper implements Serializable {
+
+    private static final long serialVersionUID = -2520775127453216350L;
+
+    private final LanguageAPI languageAPI = APILocator.getLanguageAPI();
+    private final ContentletAPI contentletAPI = APILocator.getContentletAPI();
+
+    private static final boolean RESPECT_FE_ROLES = Boolean.TRUE;
+
+    /**
+     * Private constructor
+     */
+    private ContentRelationshipsHelper() {
+
+    }
+
+    /**
+     * Provides a singleton instance of the {@link ContentRelationshipsHelper}
+     */
+    private static class SingletonHolder {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)